src/workflow.ts

```diff
@@ -80,6 +80,143 @@ export async function startWorkflow() {
     process.exit(0);
   }
 
+  // copt the initialResults
```

Comment:
Lets add comment on why we copy it
```suggestion
  // copy the initialResults
```